src/components/Footer/Footer.js

```diff
@@ -1,6 +1,158 @@
-import React from "react";
+import React from 'react';
+import styled from 'styled-components';
 
-export const Footer = () => {
-  return <div></div>;
+const Footer = () => {
+  return (
+    <footer>
+      {/* 푸터 윗부분 */}
+      <FooterUpper>
+        <ContentsWrapper>
+          <Mass>
+            <FirstOne>텀블벅</FirstOne>
+            <ContentEach>공지사항</ContentEach>
+            <ContentEach>채용</ContentEach>
+            <ContentEach>제휴∙협력</ContentEach>
+          </Mass>
+          <Mass>
+            <FirstOne>이용안내</FirstOne>
+            <ContentEach>헬프 센터</ContentEach>
+            <ContentEach>창작자 가이드</ContentEach>
+            <ContentEach>프로젝트 심사 기준</ContentEach>
+            <ContentEach>수수료 정책</ContentEach>
+          </Mass>
+          <Mass>
+            <FirstOne>정책</FirstOne>
+            <ContentEach>이용약관</ContentEach>
+            <ContentEach>개인정보 처리방침</ContentEach>
+          </Mass>
+          <Mass>
+            <FirstOne>App</FirstOne>
+            <ContentEach>
+              <AndroidBtn>안드로이드</AndroidBtn>
+            </ContentEach>
+            <li>
+              <IosBtn>iOS</IosBtn>
+            </li>
+          </Mass>
+        </ContentsWrapper>
+        <div className="supportCustomerWrapper">
```

Comment:
스타일드 컴포넌트를 사용하고 있는 만큼 클래스 셀렉터를 사용할 필요성이 크게 없어보이는 케이스입니다! 클래스명이 떠오른다면 그냥 그 이름을 스타일드 컴포넌트의 이름으로 써줘도 될거 같습니다.
```suggestion
        <SupportCustomerWrapper>
```